docs/getting-started.md

```diff
@@ -0,0 +1,144 @@
+# Getting Started
+
+Following the common .NET idiom, Marten supplies extension methods to quickly integrate Marten into any .NET application that uses the `IServiceCollection` abstractions to register IoC services. 
+For this reason, we recommend starting with a Worker Service template for console applications, or an ASP.NET Core template for web services.
+
+To add Marten to a .NET project, first go get the Marten library from Nuget:
+
+::: code-group
+
+```shell [.NET CLI]
+dotnet add package Marten
+```
+
+```powershell [Powershell]
+PM> Install-Package Marten
+```
+
+```shell [Paket]
+paket add nuget Marten
+```
+
+:::
+
+In the startup of your ASP&#46;NET Core or Worker Service application, make a call to `AddMarten()` to register Marten services like so:
+
+<!-- snippet: sample_StartupConfigureServices -->
+<a id='snippet-sample_startupconfigureservices'></a>
+```cs
+// This is the absolute, simplest way to integrate Marten into your
+// .NET application with Marten's default configuration
+builder.Services.AddMarten(options =>
+{
+    // Establish the connection string to your Marten database
+    options.Connection(builder.Configuration.GetConnectionString("Marten")!);
+
+    // If we're running in development mode, let Marten just take care
+    // of all necessary schema building and patching behind the scenes
+    if (builder.Environment.IsDevelopment())
+    {
+        options.AutoCreateSchemaObjects = AutoCreate.All;
+    }
+});
+```
+<sup><a href='https://github.com/JasperFx/marten/blob/master/src/AspNetCoreWithMarten/Program.cs#L14-L29' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_startupconfigureservices' title='Start of snippet'>anchor</a></sup>
+<!-- endSnippet -->
+
+See [Bootstrapping with HostBuilder](/configuration/hostbuilder) for more information and options about this integration.
+
+::: tip INFO
+The complete ASP<span/>.NET Core sample project is [available in the Marten codebase](https://github.com/JasperFx/marten/tree/master/src/AspNetCoreWithMarten)
+:::
```

Comment:
@Hawxy, it might be worth mentioning that the setup with IoC is not mandatory; I saw a few people believing that Marten won't work without that. It might be adding a simple example here.